algrtm/src/utils/validation.js

```diff
@@ -0,0 +1,21 @@
+const validateForm = (name, email, message) => {
+  const errors = {};
+
+  if (!name) {
+    errors.your_name = "Name field cannot be empty";
+  }
+
+  if (!email) {
+    errors.your_email = "Email field cannot be empty";
+  } else if (!/\S+@\S+\.\S+/.test(email)) {
+    errors.your_email = "Invalid email address";
+  }
+
+  if (!message) {
+    errors.message = "Message field cannot be empty";
+  }
```

Comment:
Email works great 🚀 but for the name and message we should use `.trim()` to check for whitespace characters and perhaps add a maximum length for them

Reply:
@TheMankel updated:)

Reply:
To limit the max length of the fields we can add such code:
```js
const validateForm = (name, email, message) => {
  const errors = {};

  if (!name.trim()) {
    errors.your_name = "Name field cannot be empty.";
  } else if (name.length > 20) {
    errors.your_name = "Name cannot exceed 20 characters.";
  }

  if (!email) {
    errors.your_email = "Email field cannot be empty.";
  } else if (email.length > 320) {
    errors.your_email = "Email cannot exceed 320 characters.";
  } else if (!/\S+@\S+\.\S+/.test(email)) {
    errors.your_email = "Invalid email address.";
  }

  if (!message.trim()) {
    errors.message = "Message field cannot be empty.";
  } else if (message.length > 1000) {
    errors.message = "Message cannot exceed 1000 characters.";
  }

  return errors;
};

export default validateForm;
```